README.mkd

```diff
@@ -14,7 +14,7 @@ This parser provide same feature for following languages.
 
 * Vim script
 * Python
-* Javascript 
+* JavaScript
 
 ## Example
 
```

Comment:
Should the L24 below updated like this?

before: 
```vim
let s:vimlparser = vimlparser#import()
```

after:
```vim
let s:V = vital#yourpluginname#new()
let s:vimlparser = s:V.import('VimLParser')
```

with vitalize

Reply:
👍 for writing vital API instead of autoload API.

And your "after" code should be:

```vim
let s:vimlparser = vital#yourpluginname#import('VimLParser').import()
```

According to https://github.com/vim-jp/vim-vimlparser/pull/74/files#diff-ec29e185fa61087ecf991ff8811a8acaR6

Reply:
In the first place, current example code is wrong...

```diff
-let r = s:StringReader.new(code)
-let p = s:VimLParser.new()
-let c = s:Compiler.new()
+let r = s:vimlparser.StringReader.new(code)
+let p = s:vimlparser.VimLParser.new()
+let c = s:vimlparser.Compiler.new()
```

Reply:
:sushi: